### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by prokoudine**
Mostly removing the unnecessary title case — that's not how Russian typography works :)

### CODE REVIEW

File: res/locales/ru_RU.po [hunk 1]
```diff
@@ -303,7 +306,7 @@ msgid ""
 "Constrain -> On Point before constraining tangent."
 msgstr ""
 "Дуга и отрезок должны быть соединены. Соедините их крайние точки с помощью "
-"'Ограничения -> Точка на Примитиве' перед тем, как применять ограничение "
+"'Ограничители -> Точка на примитиве' перед тем, как применять ограничители "
```
Comment (ruevs) [inline res/locales/ru_RU.po:309]: Are you sure about this?

The term "constraint" in the context that is used in SolveSpace is "ограничение". On the other hand "ограничитель" is "limiter" and does not fit well in my opinion. But Russian is not my native language.

Is there a precedent in some other CAD software that has Russian UI?

This applies to all the other places where you used "ограничители" - I'll not mark them all.

File: res/locales/ru_RU.po [hunk 2]
```diff
@@ -715,47 +718,47 @@ msgstr "&Файл"
...
 
 #: graphicswin.cpp:49
 msgid "Export 2d &View..."
-msgstr "Экспортировать &2d Вид..."
+msgstr "Экспортировать &2D-вид…"
 
 #: graphicswin.cpp:50
 msgid "Export 2d &Section..."
-msgstr "Экспортировать 2d Се&чение..."
+msgstr "Экспортировать 2D-се&чение…"
```
Comment (ruevs) [inline res/locales/ru_RU.po:749]: 2d

File: res/locales/ru_RU.po [hunk 3]
```diff
@@ -847,143 +850,143 @@ msgstr "От&далить"
...
 
 #: graphicswin.cpp:121
 msgid "Link / Assemble..."
-msgstr "&Импорт Детали / Сборка..."
+msgstr "&Импорт детали / Сборка…"
 
 #: graphicswin.cpp:122
 msgid "Link Recent"
-msgstr "Последние &Детали"
+msgstr ""
```
Comment (ruevs) [inline res/locales/ru_RU.po:961]: Why did you remove:
`msgstr "Последние &детали"`
here?

File: res/locales/ru_RU.po [hunk 4]
```diff
@@ -847,143 +850,143 @@ msgstr "От&далить"
...
 
 #: graphicswin.cpp:125
 msgid "In &Workplane"
-msgstr "В &Рабочей Плоскости"
+msgstr "В &рабочей плоскости"
 
 #: graphicswin.cpp:126
 msgid "Anywhere In &3d"
-msgstr "Режим &3d"
+msgstr "Режим &3D"
```
Comment (ruevs) [inline res/locales/ru_RU.po:973]: 3d

File: res/locales/ru_RU.po [hunk 5]
```diff
@@ -1011,43 +1014,43 @@ msgstr "И&зображение"
...
 
 #: graphicswin.cpp:150
 msgid "Other S&upplementary Angle"
-msgstr "Переключить Сме&жный Угол"
+msgstr "Переключить сме&жный угол"
 
 #: graphicswin.cpp:151
 msgid "Toggle R&eference Dim"
-msgstr "Переключить Режим Размера Для Спра&вок"
+msgstr ""
```
Comment (ruevs) [inline res/locales/ru_RU.po:1053]: Do not remove. Use:
`msgstr "Переключить режим размера для спра&вок"`

File: res/locales/ru_RU.po [hunk 6]
```diff
@@ -1474,11 +1477,11 @@ msgstr "контур имеет самопересечения!"
 
 #: groupmesh.cpp:727
 msgid "zero-length edge!"
-msgstr "вырожденный отрезок!"
+msgstr "ребро нулевой длины!"
```
Comment (ruevs) [inline res/locales/ru_RU.po:1480]: `msgstr "отрезок нулевой длины!"`
to be consistent with the rest of the UI.

File: res/locales/ru_RU.po [hunk 7]
```diff
@@ -1527,47 +1530,47 @@ msgstr "Невозможно разделить пересекаемые при
...
 
 #: mouse.cpp:603
 msgid "Style Info"
-msgstr "Настройки Стиля"
+msgstr "Настройки стиля"
 
 #: mouse.cpp:623
 msgid "Select Edge Chain"
-msgstr "Выделить Последовательность Примитивов"
+msgstr "Выделить последовательность граней"
```
Comment (ruevs) [inline res/locales/ru_RU.po:1553]: The original is better here, in my opinion. If I'm not mistaken "грань" means "face"/"facet" and is not what is intended.

File: res/locales/ru_RU.po [hunk 8]
```diff
@@ -1527,47 +1530,47 @@ msgstr "Невозможно разделить пересекаемые при
...
 
 #: mouse.cpp:629
 msgid "Toggle Reference Dimension"
-msgstr "Переключить Режим Размера Для Справок"
+msgstr ""
```
Comment (ruevs) [inline res/locales/ru_RU.po:1557]: Restore translation.

File: res/locales/ru_RU.po [hunk 9]
```diff
@@ -2254,7 +2257,7 @@ msgstr "Создать группу в новой рабочей плоскос
 
 #: toolbar.cpp:82
 msgid "New group in 3d"
-msgstr "Новая группа в 3d"
+msgstr "Новая группа в 3D"
```
Comment (ruevs) [inline res/locales/ru_RU.po:2260]: 3d

**Review by ruevs — CHANGES_REQUESTED** [review]
Hello Alexandre,

Thank you for updating the translation. It's nice to see that you are still following SolveSpace considering that the [first article about the open source release in 2013](https://web.archive.org/web/20200219182544/http://libregraphicsworld.org/blog/entry/solvespace-released-under-gpl) is yours :-)

I've added some comments inline. Please take a look.

File: res/locales/ru_RU.po (hunk 1 above)
Comment (prokoudine) [reply res/locales/ru_RU.po:309]: Fair point. Let's revert that.

File: res/locales/ru_RU.po [hunk 10]
```diff
@@ -715,47 +718,47 @@ msgstr "&Файл"
...
+msgstr "Открыть н&едавние файлы"
 
 #: graphicswin.cpp:45
 msgid "&Save"
 msgstr "&Сохранить"
 
 #: graphicswin.cpp:46
 msgid "Save &As..."
-msgstr "Сохранить &Как..."
+msgstr "Сохранить &Как…"
```
[the remark this answers is not in the document]
Comment (prokoudine) [reply res/locales/ru_RU.po:737]: Yes, it's a rather common recommendation in i18n teams

File: res/locales/ru_RU.po [hunk 11]
```diff
@@ -715,47 +718,47 @@ msgstr "&Файл"
...
 
 #: graphicswin.cpp:48
 msgid "Export &Image..."
-msgstr "Экспортировать И&зображение..."
+msgstr "Экспортировать и&зображение…"
 
 #: graphicswin.cpp:49
 msgid "Export 2d &View..."
-msgstr "Экспортировать &2d Вид..."
+msgstr "Экспортировать &2D-вид…"
```
[the remark this answers is not in the document]
Comment (prokoudine) [reply res/locales/ru_RU.po:745]: Oh well, okay :)

File: res/locales/ru_RU.po [hunk 12]
```diff
@@ -1091,63 +1094,63 @@ msgstr "Перпендикул&ярность"
...
 
 #: graphicswin.cpp:181
 msgid "Step &Dimension..."
-msgstr "Плавное Из&менение Размера..."
+msgstr "Плавное из&менение размера…"
 
 #: graphicswin.cpp:183
 msgid "&Help"
-msgstr "&Помощь"
+msgstr "&Справка"
```
[the remark this answers is not in the document]
Comment (prokoudine) [reply res/locales/ru_RU.po:1153]: Yep

File: res/locales/ru_RU.po [hunk 13]
```diff
@@ -2089,7 +2092,7 @@ msgstr ""
 "\n"
 "Для дополнительной информации посетите сайт: http://solvespace.com/\n"
 "\n"
-"SolveSpace - это свободное программное обеспечение: поощряется изменение, "
+"SolveSpace — свободное программное обеспечение: поощряется изменение, "
```
[the remark this answers is not in the document]
Comment (prokoudine) [reply res/locales/ru_RU.po:2095]: That's Russian typography rules

File: res/locales/ru_RU.po (hunk 7 above)
Comment (prokoudine) [reply res/locales/ru_RU.po:1553]: My bad, it's "Выделить последовательность ребер", of course. Unless SolveSpace absolutely needs edges to be "отрезки", in which case it's "Выделить последовательность отрезков". But I looked up the German translation, and it says "Kantenverlauf", so it definitely refers to edges.

File: res/locales/ru_RU.po [hunk 14]
```diff
@@ -1011,43 +1014,43 @@ msgstr "И&зображение"
...
 
 #: graphicswin.cpp:143
 msgid "Split Curves at &Intersection"
-msgstr "Ра&збить Кривые Пересечением"
+msgstr "Ра&збить кривые пересечением"
 
 #: graphicswin.cpp:145
 msgid "&Constrain"
-msgstr "&Ограничения"
+msgstr "&Ограничитель"
```
Comment (ruevs) [inline res/locales/ru_RU.po:1029]: "&Ограничения"

File: res/locales/ru_RU.po [hunk 15]
```diff
@@ -1577,11 +1580,11 @@ msgstr ""
 
 #: mouse.cpp:713
 msgid "Toggle Construction"
-msgstr "Переключить Режим 'Дополнительные Построения'."
+msgstr "Переключить режим «Дополнительные построения»"
 
 #: mouse.cpp:729
 msgid "Delete Point-Coincident Constraint"
-msgstr "Удалить Ограничение Совпадения Точек"
+msgstr "Удалить ограничитель совпадения точек"
```
Comment (ruevs) [inline res/locales/ru_RU.po:1587]: "oграничение"

**Review by ruevs — CHANGES_REQUESTED** [review]
Thanks for the update! I "resolved" all the topics you fixed.

Только четыре "ограничител[ь/и]" остались ;-)

File: res/locales/ru_RU.po (hunk 7 above)
Comment (ruevs) [reply res/locales/ru_RU.po]: Now it is good.

**Comment by prokoudine** [comment]
If I squash commits, will it play well with the PR?

**Comment by prokoudine** [comment]
> Or I can do it when merging.

Ah, please do :)

**Review by ruevs — APPROVED** [review]

**Comment by ruevs** [comment]
Merged. Thanks @prokoudine.

**Pull request merged**